README.md

```diff
@@ -0,0 +1,27 @@
+# Funny Blocks
+This module adds several random, funny and goofy blocks to make [Terasology](https://github.com/MovingBlocks/Terasology) even more awesome!
+
+![Funny Blocks](https://s13.postimg.org/ee79vl6mv/2018-01-08_6.png)
+From left to right:
+
+- BreakingBlock - Begins to break when the player stands on top of it
+- BluePortalBlock - Teleports the player to an activated Orange Portal Block when they stand on top of it (Press 'e' to activate)
+- OrangePortalBlock: Teleports the player to an activated Blue Portal Block when they stand on top of it (Press 'e' to activate)
+- BouncyBlock - Automatically bounces the player towards the sky when they stand on top of it
+- TrampolineTile - Enables the player to bounce off of it towards the sky
+- SpeedBlock - Briefly increases the speed of the player when they walk over it
+- SpeedBoost - Launches the player in the direction of the arrow when they walk over it
+- SpeedBooster - The Speed Boost block but in a smaller, tile format
+- CheeseWheel - Just a cheese wheel, because why not?
+- SlicedCheeseWheel - A cheese wheel with a slice taken out of it. Yum!
+- BowlingPin - A tenpin bowling pin
+
+## Playing
+After installing and activating this module in a new world you have two options as to how to access the Funny Blocks:
+1. Install and activate a world generator that includes some of these blocks (see [FunnyGenerator](https://github.com/NicholasBatesNZ/FunnyGenerators) as an example). The blocks will be spawned in the world automatically.
+2. Press the \` key and enter the command `give BlockName` where "BlockName" is the name of the desired block as taken from the list above. This block will appear in your inventory.
```

Comment:
change \` to `~` (even though ~ is the secondary character on the key, the key is commonly known as the tilde key)